### ISSUE TITLE
pip installation

### ISSUE THREAD

**Issue by AidanRocke** [openworm/behavioral_syntax#17]
I need to structure the library so that it can be installed via pip. Instructions can be found here: http://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#requirements-for-packaging-and-distributing

**Comment by MichaelCurrie** [comment]
The Open Worm Analysis Toolbox has the same open issue.  I did close it for the WCON parser, and I wrote the details of what to do here:

[](https://github.com/openworm/open-worm-analysis-toolbox/issues/139)

Obviously you only need to worry about this after you have a stable release.

Also I would think about not making your top-level folder "behavioral_syntax" the package, but instead go one subfolder in.  That's the process I've followed at Open Worm Analysis Toolbox, anyway.
